content/applications/services/project/project_management.rst

```diff
@@ -0,0 +1,95 @@
+
+.. admonition:: Did you know?
+
+   The word **Kanban** comes from Japanese and refers to the "visual board" management method.
+
+.. seealso::
+   `Odoo Tutorials: Kanban Project Management
+   <https://www.odoo.com/slides/slide/kanban-project-management-1664>`_
+
+Configuration
+=============
+
+Open the **Project** app and click :guilabel:`Create` to start a new project. Enter a
+:guilabel:`Name` for your project and click :guilabel:`Create Project`.
+
+You can customize your existing **projects** from the dashboard by clicking the drop-down toggle
+button (:guilabel:`⋮`) on your project's **card**.
+
+.. image:: project_management/project-settings.png
+   :align: center
+   :alt: Project card
+
+This enables a new menu divided into four parts:
+
+- **View**: see an overview of your project's components, such as its tasks, milestones and project
+  updates. Depending on which apps you have activated, more options may be available, such as
+  **Documents**: all uploaded files can be found under this menu, as well as in the **Documents**
+  app, under :guilabel:`Projects`;
+
+- **Reporting**: analyze your project's progress and profitability through graphics and statistics;
+
+- **Color**: make a line of color appear on the left side of the card so that your project is more
+  recognizable;
+
+- **Settings**: you can change the following:
+
+  - the :guilabel:`Name` of the project;
+  - the :guilabel:`Name of the tasks` found under that project.
+  - the :guilabel:`Customer` for whom the project is intended;
+  - the :guilabel:`Tags` used for filtering;
+  - the :guilabel:`Company` responsible for the project;
+  - the employee designated as :guilabel:`Project Manager`;
+  - the :guilabel:`Planned Date` of the project;
+  - the total :guilabel:`Allocated Hours` for that project.
+
+Additionally, you can mark the project as a :guilabel:`Favorite`, allowing you to find it using the
+:guilabel:`My Favorites` filter on the Kanban view;
+
+.. image:: project_management/project-settings-opened.png
+   :align: center
+   :alt: Project settings
+
+.. seealso::
+   `Odoo Tutorials: Customize your project
+   <https://www.odoo.com/slides/slide/customize-your-project-1662?fullscreen=1>`_
+
+Depending on the apps you have activated, other options are available:
+
+- **Sales orders**: you can view related :abbr:`SO (Sales Order)` for that project under the
+  :guilabel:`Invoicing` tab by activating the **Invoicing** app;
+
+- **Billable**: you can set the project as :guilabel:`Billable` by activating the **Invoicing** or
+  **Accounting** apps;
+
+- **Log time**: you can log time on tasks by activating the **Timesheet** app;
+
+- **Plan resources**: you can plan resources on tasks by activating the **Planning** app;
+
+- **Analytic account**: you can set an :guilabel:`Analytic Account` for that project if you have
+  the **Accounting** app activated as well as the *Analytic Accounting* option in the
+  configurations.
```

Comment:
weird to talk about it in the end when it's the first setting you see in the settings tab,
also, you don't talk about "visibility", but I think you do not need to activate any app to do it (same for milestones etc.)
So I suggest dividing this in two: the settings available in the settings tab independently of the apps activated, and the settings that depend on the apps activated